src/Symfony/Component/HttpKernel/CacheClearer/Psr6CacheClearer.php

```diff
@@ -18,14 +18,11 @@
  */
 class Psr6CacheClearer implements CacheClearerInterface
 {
-    private array $pools = [];
```

Comment:
We know that this change is not safe in the generic case: removing the default value from a property declaration (what CPP does) breaks child classes that don't call this class' constructor. I hope this won't hit us.

Did you have this concern in mind when preparing the PR? We should be careful here, we already broke a few releases because of this issue.

Reply:
I was not aware of this behavior, let me go through the whole PR again with this in mind